test/pkg/environment/common/environment.go

```diff
@@ -43,6 +43,8 @@ type ContextKey string
 
 const (
 	GitRefContextKey = ContextKey("gitRef")
+	TestingCluster   = "testing.karpenter.sh/cluster"
```

Comment:
`test.DiscoveryLabel` lives in `test/metadata` so should we move these labels into that file as well so that they can be referenced by `test.ClusterLabel`, `test.TypeLabel`, `test.DiscoveryLabel`

Reply:
PR for this change https://github.com/aws/karpenter-core/pull/431

Reply:
Buddy PR for karpenter https://github.com/aws/karpenter/pull/4361